src/LinearAlgebra/COO_Matrix.hpp

```diff
@@ -469,44 +492,60 @@ inline void COO_Matrix<ScalarT, Intdx>::permutationSizeMap(std::vector<Intdx> ro
 
     for (int i = 0; i < this->values_.size(); i++)
     {
-        if (row_perm[this->row_indexes_[i]] == -1 || col_perm[this->column_indexes_[i]] == -1)
+        if (row_perm[this->row_indices_[i]] == -1 || col_perm[this->column_indices_[i]] == -1)
         {
             this->values_[i] = 0;
         }
         else
         {
-            this->row_indexes_[i] = row_perm[this->row_indexes_[i]];
-            this->column_indexes_[i] = col_perm[this->column_indexes_[i]];
+            this->row_indices_[i] = row_perm[this->row_indices_[i]];
+            this->column_indices_[i] = col_perm[this->column_indices_[i]];
         }
     }
     this->sorted_ = false;
 }
 
 /**
- * @brief Turn matrix into the zero matrix. Does not actual delete memory
+ * @brief Turn matrix into the zero matrix. Does not actually delete memory
+ * 
+ * @SR - If it's not deleteing memory, what's the point?
  * 
  * @tparam ScalarT 
  * @tparam Intdx 
+ * 
  */
 template <class ScalarT, typename Intdx>
 inline void COO_Matrix<ScalarT, Intdx>::zeroMatrix()
 {
     //resize doesn't effect capacity if smaller
-    this->column_indexes_.resize(0);
-    this->row_indexes_.resize(0);
+    this->column_indices_.resize(0);
+    this->row_indices_.resize(0);
     this->values_.resize(0);
     this->sorted_ = true;
 }
 
+/**
+ * @brief Turn matrix into the identity matrix
+ * 
+ * @tparam ScalarT
+ * @tparam Intdx
+ * 
+ * @param[in] n size of the identity matrix
+ * 
+ * @post this = I_n
+ * 
+ * @SR - it might be better to explicitly zero out the matrix and require to be so in preconditions
+ */
```

Comment:
`@SR` is not a Doxygen markup :). Use `@todo` instead.
```suggestion
 * @todo - it might be better to explicitly zero out the matrix and require to be so in preconditions
 */
```